src/functions/asyncCredential/asyncCredentialHandler.test.ts

```diff
@@ -174,6 +120,13 @@ describe("Async Credential", () => {
         const event = buildRequest();
         const result = await lambdaHandler(event, dependencies);
 
+        expect(mockLogger.getLogMessages()[0].logMessage.message).toBe(
+          "ENVIRONMENT_VARIABLE_MISSING",
```

Comment:
mixed feelings about this - it is 'good enough' for me. Could update this to say ENVIRONMENT_VARIABLE_INVALID - it's a shared log message and keen not to split this validation out if possible.

Reply:
If you don't want to split the message and have one shared ENVIRONMENT_VARIABLE_INVALID message, we could also portentially log the value of the envVar, so when someone sees that's invalid log they can just look what the value was? Even this might be overkill as they'll already see what variable was invalid and can probs check and then work it out from there...

Will leave it up to you, I am happy with all options here

Reply:
I'm not too keen on logging it out if possible unless we know we need to see it. We're the ones in control of it, so I'm happy enough with this approach

Reply:
If it was an external team populating this value, I'd propose logging the value